app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,34 @@
+class PagesController < ApplicationController
+  def home
+  end
+    
+  def index 
+    @journal_entry = JournalEntry.all
+  end
+
+  def new_entry_form
+    @entry = JournalEntry.new
+  end
+
+  def entries;end
```

Comment:
what is the purpose of this method here? if not needed lets delete this method

Reply:
i display the pages by calling the method, if not i would have to hardcode the file link i guess.